Mathlib/RingTheory/Congruence/Opposite.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2022 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+
+import Mathlib.RingTheory.Congruence.Basic
+import Mathlib.GroupTheory.Congruence.Opposite
+
+/-!
+# Congruences on the opposite ring
+
+This file defines the order isomorphism between the congruences on a ring `R` and the congruences on
+the opposite ring `Rᵐᵒᵖ`.
+
+-/
+
+variable {R : Type*} [Add R] [Mul R]
+
+namespace RingCon
+
+/--
+If `c` is a `RingCon R`, then `(a, b) ↦ c b.unop a.unop` is a `RingCon Rᵐᵒᵖ`.
```

Comment:
Any idea why we swap the argument order?

Reply:
Ah, I guess this question is better directed at https://github.com/leanprover-community/mathlib4/pull/13610/files#r2032230268